docs/configuration/dtable-events-conf.md

```diff
@@ -75,17 +70,6 @@ SeaTable runs this task every hour to send base email notifications for base upd
 | --------- | ------------------------------------------------------------ | ------- |
 | `enabled` | Enables or disables the email notifications for base updates | true    |
 
-### `[NOTIFY-SCANNER]`
```

Comment:
@wacmkxiaoyi Can you provide some more information why this section was removed? Same for `ROWS-COUNTER`?